fearlessTests/Modules/ChainAccountBalanceList/ChainAccountBalanceListTests.swift

```diff
@@ -52,12 +55,26 @@ class ChainAccountBalanceListTests: XCTestCase {
                     stateCompletionExpectation.fulfill()
                 }
             }
+            
+            stub.didReceive(locale: any()).thenDoNothing()
+        }
+        
+        let assetRepository = SubstrateDataStorageFacade.shared.createRepository(
+            mapper: AnyCoreDataMapper(AssetModelMapper())
+        )
+        var assets: [AssetModel] = []
```

Comment:
Why not `let assets = chains.reduce([]) { $0 + $1.assets }.map { $0.asset }` ? Instead of for/for accumulation